polaris-react/src/components/ChoiceList/ChoiceList.scss

```
@@ -1,50 +1,5 @@
 @import '../../styles/common';
 
-.ChoiceList {
-  margin: 0;
-  padding: 0;
-  border: none;
-}
-
-.ChoiceItem:not(:last-child) {
-  @media #{$p-breakpoints-md-down} {
-    margin-bottom: var(--p-space-4);
-  }
-}
-
-.titleHidden > .Title {
-  @include visually-hidden;
-}
-
-.Choices {
-  margin: 0;
-  padding: 0;
-  list-style: none;
-}
-
 .ChoiceChildren {
-  @media #{$p-breakpoints-md-down} {
-    margin-top: var(--p-space-4);
-  }
-  margin-bottom: var(--p-space-2);
   padding-left: calc(var(--p-space-2) + var(--p-choice-size));
```

```suggestion
  padding-left: calc(var(--p-space-2) + var(--p-space-5));
```

Looks like `--p-choice-size` is a `20px` legacy token that we already have in spacing tokens.

---

https://github.com/Shopify/polaris/blob/main/polaris-react/src/components/Choice/Choice.scss#L47

`--p-choice-size` changes based on breakpoint so can't be replaced with the spacing token.

---

Ahh, missed that line thank you for the context!